### ISSUE TITLE
[Base] Failure on startup with build 1.4.7-49

### ISSUE THREAD

**Issue by SpoonsJTD** [keepcalm/BukkitForge#265]
You might be in the middle of fixes, keepcalm, sorry if so. But if not, I am now getting a bukkitforge crash with 1.4.7-49:

```
2013-01-19 03:51:27 [SEVERE] [ForgeModLoader] Something real bad happened! The B
ukkitAPI will not be running for this Minecraft session.
You should *probably* report this bug to the developer(s) at https://github.com/
keepcalm/BukkitForge/issues
java.lang.RuntimeException: BukkitForge was packaged incorrectly - sorry.
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.j
ava:82)
        at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.
java:305)
        at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager
.java:230)
        at keepcalm.mods.bukkit.bukkitAPI.BukkitServer.loadPlugins(BukkitServer.
java:912)
        at keepcalm.mods.bukkit.bukkitAPI.BukkitServer.<init>(BukkitServer.java:
273)
        at keepcalm.mods.bukkit.asm.BukkitStarter.run(BukkitStarter.java:46)
        at java.lang.Thread.run(Unknown Source)
```

I've removed all 'ported' mods. Started fine before the recent flurry of changes.

**Comment by ghost** [comment]
have same issue - server start only with empty plugins directory

**Comment by SpoonsJTD** [comment]
The code is looking for files vcb2obj.srg and cb2pkgmcp.srg as resources. I'm not smart enough to configure the build to include those files in the root directory of the jar file, but until keepcalm does, you can work around this. Rename the bukkitforge jar to zip and drag the two files from the src\porting folder from the source into the zip after first renaming vcb2obf_1.4.7.srg to vcb2obj.srg. Rename the zip back to jar and you should be good to go.

**Comment by SpoonsJTD** [comment]
Fixed with 1.4.7-50.
